src/components/card/Card/Card.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import classnames from 'classnames'
+
+import { GridLayoutProp, applyGridClasses } from '../../grid/Grid/Grid'
+
+interface CardProps {
+  layout: 'standardDefault' | 'flagDefault' | 'flagMediaRight'
```

Comment:
just a nitpick while I'm testing out adding some more examples to Storybook, could we make `standardDefault` the default value for layout?